shared/misc/ansible_galaxy_meta_template.yml

```diff
@@ -0,0 +1,30 @@
+galaxy_info:
+  author: SCAP Security Guide development team
+  description: @DESCRIPTION@
+
+  issue_tracker_url: https://github.com/OpenSCAP/scap-security-guide/issues
+
+  license: BSD-3-Clause
+
+  min_ansible_version: 1.2
```

Comment:
Great catch! Fixed. I made it into a central variable in ssgcommon so that there is less risk of it being desynchronized between docs, pre_tasks and the meta yml.